model/jpa/src/main/resources/META-INF/orm.xml

```diff
@@ -48,7 +48,8 @@ http://java.sun.com/xml/ns/persistence/orm_2_0.xsd"
             </many-to-one>
             <many-to-many name="categories" fetch="EAGER">
                 <cascade>
-                    <cascade-all/>
+                    <cascade-persist/>
+                    <cascade-merge/>
                 </cascade>
```

Comment:
@matzew I think I found this bug when using a real database instead of an in memory one
@sebastienblanc merge is when updating the entity don't know if we do that for installations, guess it doesn't hurt to put that one there as well.